src/extension/app-schema/app-schema-mongo-test/src/test/resources/data/stations1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "1",
+  "name": "station 1",
```

Comment:
Yeah no sure what happen, I copy paste test cases to doc :disappointed_relieved:.